contrib/cf-remote/cf_remote/packages.py

```diff
@@ -165,5 +167,5 @@ def pick_version(self, version):
                 return Release(release)
 
     def __str__(self):
-        lines = [str(x) for x in self.releases]
-        return "\n".join(lines)
+        releases = [str(x.version) for x in self.releases]
+        return ", ".join(releases)
```

Comment:
You can combine this to `", ".join(str(x.version) for x in self.releases)`.

Reply:
Fixed.